lib/ui/fixtures/ui_test.dart

```diff
@@ -1,4 +1,3 @@
-// @dart = 2.6
```

Comment:
can you add a README to the fixtures directory that explains what it's for? (i'm assuming it's for development-time resources that don't end up in production binaries, but it's not clear exactly how we enforce that or why they are here rather than, say, in the `testing` directory)

Reply:
These directories exist in several places - the way that the C++ unit test setups work, the fixtures directory contains a Dart file and sometimes other data files like images for testing.

See also for example //shell/fixtures/shell_test.dart.

There's a GN template that defines how this works called `test_fixtures` that has a bunch of commentary on it: https://github.com/flutter/engine/blob/master/testing/testing.gni#L230-L250

Reply:
That commentary never actually says what it means by "fixtures". I'm also not sure how I'd ever find that if I started from a fixtures directory. Having some breadcrumbs would really help people orient themselves when first being exposed to this stuff, I think.

Reply:
Added a readme explaing a bit and pointing to the BUILD file and the template.